### DISCUSSION TITLE
Applicability of MOOSE electrical module in PLASMA flow control

### DISCUSSION THREAD

**Discussion by Badtreater** [idaholab/moose#24885] [Q&A Modules: General]
Dear researchers and developers, 
thank you guys so much for providing the open-source PLASMA simulation codes. I am a novice in PLASMA research and working hard to complete my doctoral studies.However, due to my poor ability on programming of finite element solvers, it has been hard to complete the simulation code of PLASMA completely on my own. To be honest, I was very excited when I first saw Zapdos. I thought it might help me to supplement my lack of simulation.
To get back to the point, I hope to use Zapdos to complete the simulation work in the literature[1], but unfortunately, due to a lack of understanding of the underlying solver, I am unable to make a judgment on whether Zapdos can correctly complete this simulation process. I am eager to receive your answers on the applicability of the Zapdos underlying solver to the simulation of PLASMA flow control actuators. If the work can proceed, I also hope to include the reference of Zapdos in my own paper when publishing articles in the future. Thank you so much. The reference is attached in the email.

Reference
[1]Kunwar Pal Singh, Subrata Roy; Modeling plasma actuators with air chemistry for effective flow control. Journal of Applied Physics 15 June 2007; 101 (12): 123308. https://doi.org/10.1063/1.2749467

**Comment by lindsayad** [comment]
Hi @Badtreater as your question is more about Zapdos, can you open a discussion on the Zapdos repository? https://github.com/shannon-lab/zapdos/discussions
